osu.Game.Tests/Visual/Online/TestSceneNewsOverlay.cs

```diff
@@ -1,66 +1,27 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using System;
-using osu.Framework.Graphics;
 using osu.Game.Overlays;
-using osu.Game.Overlays.News;
 
 namespace osu.Game.Tests.Visual.Online
 {
     public class TestSceneNewsOverlay : OsuTestScene
     {
-        private TestNewsOverlay news;
+        protected override bool UseOnlineAPI => true;
```

Comment:
I would rather avoid having more scenes that rely on online API and instead use the dummy API to mock requests. We could have an online scene too but to me its usefulness is doubtful anyway as there aren't any asserts here at all.